### PR TITLE
[MIG][13.0] sale_report_margin: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21674

### CODE REVIEW

File: sale_report_margin/i18n/sale_report_margin.pot [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* sale_report_margin
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```
Comment (manuelcalerosolis) [inline sale_report_margin/i18n/sale_report_margin.pot]: "Project-Id-Version: Odoo Server 13.0\n"

File: sale_report_margin/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+* Sergio Teruel <sergio.teruel@tecnativa.com>
```
Comment (manuelcalerosolis) [inline sale_report_margin/readme/CONTRIBUTORS.rst]: * `Tecnativa <https://www.tecnativa.com>`__:

  * Sergio Teruel
  * David Vidal

**Review by manuelcalerosolis — APPROVED** [review]
Code and functional review.

**Comment by sergio-teruel** [comment]
@manuelcalerosolis changes done!!

**Review by Tardo — APPROVED** [review]
:+1: LGTM, tested on runbot

**Pull request merged**